example/docker-compose/local/readme.md

```diff
@@ -2,7 +2,12 @@
 In this example all data is stored locally in the `tempo-data` folder. Local storage is fine for experimenting with Tempo
 or when using the single binary, but does not work in a distributed/microservices scenario.
 
-1. First start up the local stack.
+Before starting, login to ghcr so you can access the images, using something like:
```

Comment:
do you really need to be logged in to pull from ghcr.io? I don't think I'm logged in and I can pull the image without issue.

Using [this stackoverflow answer](https://stackoverflow.com/questions/36022892/how-to-know-if-docker-is-already-logged-in-to-a-docker-registry-server):

```
> cat ~/.docker/config.json
{
	"auths": {
		"https://index.docker.io/v1/": {}
	},
...
```

I don't seem to have an entry for ghcr.io

Reply:
You're right. I've just run this on another machine and it works. Do you think it's worth leaving in the permissions change, or just bin the whole PR ?

Reply:
I think you have some good updates in the doc itself. You've added some clarifications that help people troubleshoot. If these updates aren't specifically needed, you can keep the section and label it as a "If you want to log in to GHCR" section.

What would you like to do?